CHANGELOG.md

```diff
@@ -48,6 +48,7 @@ This is the log of notable changes to the Expo client that are developer-facing.
 - added support for picking animated GIFs with `ImagePicker` by [@sjchmiela](https://github.com/sjchmiela) ([#3844](https://github.com/expo/expo/pull/3844))
 - added support for headers in `downloadAsync` method in `FileSystem` by [@mczernek](https://github.com/mczernek) ([#3911](https://github.com/expo/expo/pull/3911))
 - added support for custom poster styles in `Video` by [@sjchmiela](https://github.com/sjchmiela) ([#4165](https://github.com/expo/expo/issues/4165))
+- added `pausesUpdatesAutomatically` option to automatically pause background location updates based on the `activityType` ([#4167](https://github.com/expo/expo/pull/4167) by [@tsapeta](https://github.com/tsapeta))
```

Comment:
```suggestion
- added `pausesUpdatesAutomatically` option to automatically pause background location updates based on the `activityType` by [@tsapeta](https://github.com/tsapeta) ([#4167](https://github.com/expo/expo/pull/4167))
```

Reply:
That's right 😅
Anyway imho `(#PR by @someone)` is better than what we do right now. Author of the pull request is not really important here or at least not more important than the pull request which is inside parentheses that reduce its significance. Would you mind if we change to this template in the future versions?